roles/multi-tenancy/tasks/onboard-offboard-cas-servers.yaml

```diff
@@ -52,74 +52,11 @@
   tags:
     - offboard
 
-# Delete the site.yaml file
-- name: delete site.yaml
+# Delete the sasdeployment.yaml file
+- name: delete sasdeployment.yaml
   file:
     state: absent
-    path: "{{ DEPLOY_DIR }}/site.yaml"
+    path: "{{ DEPLOY_DIR }}/{{ NAMESPACE }}-sasdeployment.yaml"
```

Comment:
Why the need for namespacing this item? Can we have multiple sitedeployments under one deploy_dir?

Reply:
This was a recommendation from the testing team, it's namespaced to follow the same naming scheme as the SAS Documentation 
https://go.documentation.sas.com/doc/en/sasadmincdc/v_033/dplyml0phy0dkr/p0nid9gu3x2cvln1pzpcxa68tpom.htm

Reply:
This was done given they use docker in a flat directory. Do we not still create the name spaced directory and keep all the artifacts there?

Reply:
We do, this file would be under CLUSTER_NAME/NAMESPACE/, I'll ping you the internal comment from the ticket.